lib/boost_1.69.0/boost/pending/integer_log2.hpp

```diff
@@ -4,6 +4,6 @@
 #include <boost/integer/integer_log2.hpp>
 #include <boost/config/header_deprecated.hpp>
 
-// BOOST_HEADER_DEPRECATED("<boost/integer/integer_log2.hpp>");
+BOOST_HEADER_DEPRECATED("<boost/integer/integer_log2.hpp>");
```

Comment:
@syclik this seems like it was manually commented out. Do we still need to comment this out. Will try to add to the script if so.

Reply:
Ok, it seems that we do need it.